lib/puppet/configurer.rb

```diff
@@ -246,9 +267,15 @@ def run_internal(options)
   private :run_internal
 
   def send_report(report)
+    reportSendTime = thinmark {
```

Comment:
Nit picky comment: the preferred naming convention for local variables is to use `snake_case`.